README-IT.md

```diff
@@ -105,6 +106,12 @@ Se hai domande o commenti, crea un problema/issue.
 - [Le Pratiche sul COME/HOWTO rilasciare lo Software](http://en.tldp.org/HOWTO/Software-Release-Practice-HOWTO/) - Questo COME/HOWTO descrive le buone pratiche di rilascio per Linux e altri progetti open-source. Seguendo queste pratiche, renderete il più semplice possibile per gli utenti a svillupare/creare il vostro codice e usarlo, e per gli altri sviluppatori capire il vostro codice e collaborare con voi per migliorarlo.
 - [Open Sources 2.0: L'Evoluzione della Contribuzione](https://archive.org/details/opensources2.000diborich) (2005) - Open Sources 2.0 è una raccolta di saggi penetranti e stimolanti dei leader tecnologici di oggi che continua a dipingere quadro evolutivo sviluppato nel libro del 1999, Open Sources: Le voci della revoluzione.
 
+## Iniziative per contribuire all'Open Source
+- [In palio](https://up-for-grabs.net/) - un elenco di progetti con problemi per i principianti.
```

Comment:
dire in palio sembra che ssiano da vincere, direi più "disponibili"
```suggestion
- [Disponibli](https://up-for-grabs.net/) - un elenco di progetti con problemi per i principianti.
```

Reply:
ciao, in questo caso avevo lasciato "in palio" in quanto già presente e così nel README-IT. Sarebbe meglio aprire una PR per correggerlo anche negli altri punti?

Reply:
forse, o forse sono troppo puntigliosa io - non ho ancora guardato nel dettaglio il file intero, anche se avevo intenzione di farlo

Reply:
va bene, per ora aggiorno seguento il tuo suggerimento!